blog/2024-05-13-may-releases.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: "May 2024 Releases"
+slug: "2024-05-13-march-releases"
+authors: [beeankha]
+tags: [announcement, conda, conda-build]
+description: |
+  Conda 24.5.0 and conda-build 24.5.0 have been released! 🎉
+image: img/blog/2024-05-13-may-releases/banner.png
+---
+
+The May 2024 releases included updates to conda and conda-build! 🎉 Both of these have been released to both `main` and `conda-forge`.
+
+## Changes in Conda [24.5.0](https://github.com/conda/conda/releases/tag/24.5.0)
+
+To update conda, run:
+
+```bash
+conda install -n base conda=24.5.0
+```
+
+Amongst other changes, this new version of conda enables MSYS2 packages to utilize upstream installation prefixes, adds support for
+CEP-15 `base_url` field in `repodata.json`, and allows specification of channel URLs using a glob-like wildcard pattern in custom
+channel settings (_e.g._, for user with auth handler plugins). See the list below or the [release notes](https://github.com/conda/conda/releases/tag/24.5.0)
+for more details.
+
+<!-- truncate -->
+
+### ✨ What's New? ✨
+
+- Report which `MatchSpec` item caused `Invalid*Spec` exceptions for more informative error messages.
+- Remove `setuptools` remainings (`MANIFEST.in`, `wheel` build dependency) not required since the move to `hatch` in [pull request #12509](https://github.com/conda/conda/pull/12509).
+- Remove and update any imports inside conda that is importing from `conda/exports.py`.
+
+### 🔧 What Got Fixed? 🔧
+
+- Fix `conda notices --json` to correctly output JSON.
```

Comment:
```suggestion
- Fix `conda notices --json` to correctly outputs JSON.
```

Reply:
The second verb ("output") needs to stay in infinitive tense in this sentence